scripts/src/submission/submission.py

```diff
@@ -237,7 +236,6 @@ def __post_init__(self):
         if not self.modified_files:
             self.modified_files = []
             self._get_modified_files()
-            self._parse_modified_files()
```

Comment:
We no longer need to parse files here?

Also, in the next couple of lines (I can't select it here for some reason) - is there a reason why `files_api_url` is set using re.sub now?